src/pages/LivePreviewerPage/LivePreviewerPage.tsx

```diff
@@ -88,7 +93,7 @@ const LivePreviewer: VoidFunctionComponent<LivePreviewerProps> = (props) => {
   if (resume) {
     return (
       <LivePreviewContainer>
-        <LivePreviewerTemplate data={{ ...resume, id: props.match.params.id }} />
+        <LivePreviewerTemplate data={{ ...resume, id: id }} />
```

Comment:
Can also be shorted syntax 🤣 
```suggestion
        <LivePreviewerTemplate data={{ ...resume, id }} />
```